src/store.ts

```diff
@@ -236,6 +236,36 @@ export default class Store {
           on delete cascade
       )`
     )
+    await this.dbRunAsync(
+      `create table if not exists ballot_cards (
+        id varchar(36) primary key,
+        batch_id varchar(36),
+        created_at datetime default current_timestamp not null,
+
+        front_original_filename text unique,
+        front_normalized_filename text unique,
+        front_marks_json text,
+        front_cvr_json text,
+        front_metadata_json text,
+        front_adjudication_json text,
+        front_adjudication_info_json text,
```

Comment:
I wonder if adjudication info should be by card, rather than by page.

Reply:
It probably should be, I was just doing this to make minimal changes first.